test/integration/puppetlabs/services/legacy_routes/legacy_routes_test.clj

```diff
@@ -41,6 +44,11 @@
 (deftest ^:integration old-master-route-config
   (testing "The old map-style route configuration map still works."
     (bootstrap/with-puppetserver-running-with-mock-jrubies
+     "MOCK.TODO: discuss - this is *probably* ok because all we really care about
+     is that the webserver is accepting connections and handing them to the correct
+     ring handler.  Would be better if we could implement this to hit a clojure
+     endpoint instead of a Ruby one but I don't remember if there are any pure-clojure
+     endpoints under /puppet."
```

Comment:
static-file-content is a pure Clojure one, I think, although it would seem to come with a fair amount of extra overhead to setup with the versioned-code-service config, et al.  TBH, though, I think it's fine to just have this one only go so far as to ensure that the request would get handed off to the Ruby layer.  The only reason this test exists, if I recall correctly, is for backward compatibility with the one early release of 2.0 where the `:invalid-in-puppet-4` key was present in the config.  Probably pretty unlikely that many users will even have this format in their configs after upgrading to later Puppet Server versions.